test/ebanx/set_source_version_test.rb

```diff
@@ -9,6 +9,6 @@
   it "set source version succesfully" do
     ebanx.add_source_data @source, @version
     return_header = ebanx.format_user_data
-    return_header['X-Ebanx-Client-User-Agent'].must_equal "SDK-RUBY/1.6.0 TEST-SOURCE/1.2.3"
+    return_header['X-Ebanx-Client-User-Agent'].must_equal "SDK-RUBY/1.7.0 TEST-SOURCE/1.2.3"
```

Comment:
I guess It could be a patch
```suggestion
    return_header['X-Ebanx-Client-User-Agent'].must_equal "SDK-RUBY/1.6.1 TEST-SOURCE/1.2.3"
```